sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentAction.java

```diff
@@ -0,0 +1,136 @@
+        return modelVersion;
+    }
+
+    /**
+     * Set the model version. This value indicates which model will be used for scoring, e.g. "latest", "2019-10-01".
+     * If a model-version is not specified, the API will default to the latest, non-preview version.
+     *
+     * @param modelVersion The model version.
+     *
+     * @return The {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setModelVersion(String modelVersion) {
+        this.modelVersion = modelVersion;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code includeStatistics}.
+     *
+     * @return The value of {@code includeStatistics}.
+     */
+    public boolean isIncludeStatistics() {
+        return includeStatistics;
+    }
+
+    /**
+     * Set the value of {@code includeStatistics}. The default value is false by default.
+     * If set to true, indicates that the service should return document and document batch statistics
+     * with the results of the operation.
+     *
+     * @param includeStatistics If a boolean value was specified in the request this field will contain
+     * information about the document payload.
+     *
+     * @return the {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setIncludeStatistics(boolean includeStatistics) {
+        this.includeStatistics = includeStatistics;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code disableServiceLogs}.
+     *
+     * @return The value of {@code disableServiceLogs}. The default value of this property is 'false'. This means,
+     * Text Analytics service logs your input text for 48 hours, solely to allow for troubleshooting issues. Setting
+     * this property to true, disables input logging and may limit our ability to investigate issues that occur.
+     */
+    public boolean isServiceLogsDisabled() {
+        return disableServiceLogs;
+    }
+
+    /**
+     * Set the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs The default value of this property is 'false', except for methods like
+     * 'beginAnalyzeHealthcareEntities' and 'recognizePiiEntities'. This means, Text Analytics service logs
+     * your input text for 48 hours, solely to allow for troubleshooting issues. Setting this property to true,
+     * disables input logging and may limit our ability to investigate issues that occur.
+     *
+     * @return the {@link AnalyzeSentimentAction} object itself.
+     */
+    public AnalyzeSentimentAction setServiceLogsDisabled(boolean disableServiceLogs) {
+        this.disableServiceLogs = disableServiceLogs;
+        return this;
+    }
+
+    /**
+     * Get the value of {@code includeOpinionMining}. The boolean indicator to include opinion mining data in the
+     * returned result. If this flag is specified, you'll get a {@code opinions} property on SentenceSentiment.
+     * It is provided by service v3.1-preview.1 and later.
```

Comment:
should we update this to say v3.1 and later?

Reply:
I can update it since the v3.1 is more official supported version.